src/detections/configs.rs

```diff
@@ -1163,7 +1163,7 @@ pub struct JSONOutputOption {
 #[derive(Parser, Clone, Debug)]
 #[clap(
     author = "Yamato Security (https://github.com/Yamato-Security/hayabusa) @SecurityYamato)",
-    help_template = "\nHayabusa v2.5.0-dev\n{author-with-newline}\n{usage-heading}\n  hayabusa.exe <COMMAND> [OPTIONS]\n  hayabusa.exe help <COMMAND>\n\n{all-args}{options}",
+    help_template = "\nHayabusa v2.5.0 \"Golden Week Release\"\n{author-with-newline}\n{usage-heading}\n  hayabusa.exe <COMMAND> [OPTIONS]\n  hayabusa.exe help <COMMAND>\n\n{all-args}{options}",
```

Comment:
```suggestion
    help_template = "\nHayabusa v2.5.0\n{author-with-newline}\n{usage-heading}\n  hayabusa.exe <COMMAND> [OPTIONS]\n  hayabusa.exe help <COMMAND>\n\n{all-args}{options}",

```

The output differs from the output of other commands because the Golden Week Release is included only here.

Reply:
@hitenkoku Thanks for the comment/suggestion! I changed the Release message for all commands and also fixed a typo in the parentheses. Could you please check.

Reply:
thanks for your fix.
I checked it. LGTM.